client/src/client-only-routes/ShowProjectLinks.js

```diff
@@ -142,7 +150,12 @@ const ShowProjectLinks = props => {
       ({ link, title, id }) => (
         <li key={id}>
           <Link to={link} className='project-link'>
-            {title}
+            <Trans
+              i18nKey={`certification.project.title.${title}`}
+              defaults={title}
+            >
+              placeholder
+            </Trans>
```

Comment:
For some reason bracket notation was not working for dynamic title in `i18nKey`, so I went with the dot notation.